runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -189,7 +189,7 @@ public SkillXPInfo getSkillXpInfo(Skill skill)
 	}
 
 	@Subscribe
-	public void onXpChanged(ExperienceChanged event)
+	public void onXpChanged(SkillUpdate event)
```

Comment:
Probably all these onXpChanged and onExperienceChanged methos should be renamed if you are renaming the event.